packages/shopify-codemod/test/fixtures/conditional-assign-to-if-statement/assign-to-other-member.output.js

```diff
@@ -0,0 +1,3 @@
+this.foo || (this.bar = 'lol');
+self.foo || (self.bar = 'lols');
+x || (y = 'lols');
```

Comment:
That could be cleared up by having a better transform name.  `default-value-assign-to-if-statement`?